File: src/main/java/ru/trukhmanov/twochairsbackend/dto/game/DeckDto.java
package ru.trukhmanov.twochairsbackend.dto.game;

public record DeckDto(
        long id,
        String type,          // DEFAULT/PAID/USER
        String visibility,    // PRIVATE/PUBLIC
        String title,
        String description,
        int ageRating,
        int priceRub,
        boolean locked,
        Long ownerUserId
) {}

File: src/main/kotlin/ru/trukhmanov/twochairsbackend/dto/game/deck/DeckDto.kt
package ru.trukhmanov.twochairsbackend.dto.game.deck

data class DeckDto(
    val id: Long,
    val type: String,
    val visibility: String,
    val title: String,
    val description: String?,
    val ageRating: Int,
    val priceRub: Int,
    val locked: Boolean,
    val ownerUserId: Long?
)
